spring-integration-debezium/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" />
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.junit" level="warn" />
+		<Logger name="org.testcontainers" level="INFO"/>
```

Comment:
Sorry, but do we really still need these two categories to manage?

Reply:
Yes i've been using testcontainers log configuration especially when turned in debug mode.